wolfcrypt/test/test.c

```diff
@@ -674,9 +680,11 @@ int wolfcrypt_test(void* args)
     heap_baselineBytes = wolfCrypt_heap_peakBytes_checkpoint();
 #endif
 
+#ifndef NO_TEST_PRINTF
```

Comment:
Top of test.c has some macros for overriding printf`. Perhaps something like `#elif defined(NO_PRINTF #define printf(...) /* not available */`.